src/cc/libbpf.c

```diff
@@ -620,7 +620,11 @@ int bpf_prog_get_tag(int fd, unsigned long long *ptag)
   fgets(fmt, sizeof(fmt), f); // pos
   fgets(fmt, sizeof(fmt), f); // flags
   fgets(fmt, sizeof(fmt), f); // mnt_id
-  fgets(fmt, sizeof(fmt), f); // prog_type
+  fgets(fmt, sizeof(fmt), f); // prog_type or ino
```

Comment:
How about using sscanf here ? This will make the code more readable and reliable.

Reply:
Totally agree if that is considered an ok change and not too big, works for me.

Reply:
ec45eb957415c568a25853acfc25837c9bff891f